.github/workflows/continuous-integration.yml

```diff
@@ -11,23 +11,18 @@ on:
 jobs:
   phpunit:
     name: "Tests"
-    runs-on: "ubuntu-20.04"
+    runs-on: "ubuntu-latest"
 
     strategy:
       matrix:
         php-version:
-          - "7.4"
-          - "8.0"
           - "8.1"
           - "8.2"
           - "8.3"
```

Comment:
Can we enable v8.4?

Reply:
Also, we should add a `--prefer-lowest` job to test for lowest deps versions.